webpack/set-webpack-public-path-plugin/src/codeGenerator.ts

```diff
@@ -141,31 +138,18 @@ export function getSetPublicPathCode(
  * @public
  */
 export function getGlobalRegisterCode(debug: boolean = false): string {
-  const lines: string[] = [
-    '(function(){',
-    `if (!${registryVariableName}) ${registryVariableName}={};`,
-    `var scripts = document.getElementsByTagName('script');`,
-    'if (scripts && scripts.length) {',
-    '  for (var i = 0; i < scripts.length; i++) {',
-    '    if (!scripts[i]) continue;',
-    `    var path = scripts[i].getAttribute('src');`,
-    `    if (path) ${registryVariableName}[path]=true;`,
-    '  }',
-    '}',
-    '})();'
-  ];
-
-  const joinedScript: string = joinLines(lines);
-
-  if (debug) {
-    return `${EOL}${joinedScript}`;
-  } else {
-    const minifyOutput: uglify.MinifyOutput = uglify.minify(joinedScript, {
-      compress: {
-        dead_code: true
-      }
-    });
-
-    return `${EOL}${minifyOutput.code}`;
-  }
+  // Minified version of this code:
```

Comment:
If you think this will ever need to be modified, maybe throw some steps in to follow to get to the minified code? Otherwise not a big deal

Reply:
Minifying code is pretty straightforward. I'm not really worried about that.